proposals/lip-0052.md

```diff
@@ -2294,7 +2301,7 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
     - The `userSubstore` array, if the value of the `owner` property has length `LENGTH_ADDRESS` bytes, or
     - The `escrowSubstore` array, if the value of the `owner` property has length `LENGTH_CHAIN_ID` bytes.
   - The `NFTSubstore` must be in lexicographical order of `nftID`.
-    - The `attributesArray` must be in lexicographic order of `module` name.
+    - The `attributesArray` must be in lexicographic order of `module` name, and all values of `module` must be unique.
```

Comment:
I find `attributesArray` description in the table not very clear. Could it be "used to store information specific to the NFT." ?

Reply:
You mean in the table of types, i.e. this description: "Used for array of NFT attributes."?
Please confirm and I'll change it.

Reply:
Yes, exactly.

Reply:
Updated, thanks.